app/models/manageiq/providers/vmware/cloud_manager/orchestration_template.rb

```diff
@@ -104,4 +312,31 @@ def validate_format
   def self.display_name(number = 1)
     n_('vApp Template', 'vApp Templates', number)
   end
+
+  def param_name(param, group_name, extra = {})
+    group_name = Base64.encode64(group_name).strip
+    extra_attrs = extra.present? ? Base64.encode64(extra.to_yaml).strip : ''
+    "#{param}-#{group_name}___#{extra_attrs}"
+  end
+
+  def insert_vapp_networks(ovf)
```

Comment:
Ok; I understand now the problem, thanks for the clarification.

For me it is important that this can be overcome with a proper vApp template specification.